lib/jwt/encode.rb

```diff
@@ -9,7 +10,7 @@ class Encode
     attr_reader :payload, :key, :algorithm, :header_fields, :segments
 
     def self.base64url_encode(str)
-      Base64.encode64(str).tr('+/', '-_').gsub(/[\n=]/, '')
+      Base64.urlsafe_encode64(str).delete('=')
```

Comment:
This does the same thing as `Base64.urlsafe_encode64(str, padding: false)` but the padding option wasn't added until recently.